test/libsolidity/SolidityCompiler.cpp

```diff
@@ -31,7 +31,7 @@ namespace solidity
 namespace test
 {
 
-BOOST_FIXTURE_TEST_SUITE(Compiler, AnalysisFramework)
+BOOST_FIXTURE_TEST_SUITE(SolidityCompiler, AnalysisFramework)
```

Comment:
This is only to remove the annoyance of the test suite not matching the file name.